### ISSUE TITLE
Lib.FilePath.normalise throws on "./."

### ISSUE THREAD

**Issue by formulaRoot** [dfinity/motoko#3212]
Motoko 0.6.25 (source 0lx3d072-wsc9kyjw-srkxlrgy-sdfj518j)

Fatal error: exception Invalid_argument("option is None")
Raised at Stdlib.invalid_arg in file "stdlib.ml", line 30, characters 20-45
Called from Stdlib__option.get in file "option.ml" (inlined), line 21, characters 41-69
Called from Lib.FilePath.normalise in file "lib/lib.ml", line 554, characters 11-60
Called from Ic__Url.parse in file "ic/url.ml", line 84, characters 31-57
Called from Pipeline__Resolve_import.resolve_import_string in file "pipeline/resolve_import.ml", line 174, characters 8-19
Called from Stdlib__list.iter in file "list.ml", line 110, characters 12-15
Called from Pipeline__Resolve_import.resolve.(fun) in file "pipeline/resolve_import.ml", line 256, characters 4-105
Called from Diag.with_message_store in file "lang_utils/diag.ml", line 82, characters 10-13
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Pipeline.chase_imports.go in file "pipeline/pipeline.ml", line 368, characters 28-78
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Diag.traverse_ in file "lang_utils/diag.ml", line 48, characters 20-25
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Pipeline.chase_imports.go_set in file "pipeline/pipeline.ml" (inlined), line 400, characters 20-42
Called from Pipeline.chase_imports in file "pipeline/pipeline.ml", line 402, characters 47-63
Called from Pipeline.load_progs in file "pipeline/pipeline.ml", line 410, characters 21-52
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Diag.bind in file "lang_utils/diag.ml", line 32, characters 27-30
Called from Pipeline.compile_files in file "pipeline/pipeline.ml", line 687, characters 27-71
Called from Dune__exe__Moc.process_files in file "exes/moc.ml", line 194, characters 49-94
Called from Dune__exe__Moc in file "exes/moc.ml", line 288, characters 4-23

**Comment by nomeata** [comment]
Thanks! This looks as if you wrote an `import` statement that moc didn't expect. Can you share that code line?

**Comment by formulaRoot** [comment]
Sure.

` import actorName "./.";`

For context:

- I had all my code in `main.mo` so I was in the process of migrating some code  to `file2.mo` 
- Code in `file2.mo` had dependencies in `main.mo`
- My attempt to import actor from main.mo let me to that bug after I tried to deploy.

**Comment by formulaRoot** [comment]
@nomeata I confirmed that this is a bug. Yesterday, I got the import statement wrong and while trying to import. 

I just reproduced the same exact error with what I believe is the correct import statement syntax. 

Context: 
- Moving `FeatureAtypes `from `main.mo` to `featuretypes.mo`
- Added an import statement in `main.mo` to import those types from `featuretypes.mo` like so: `Import FeatureAtypes "featuretypes" `
